scripts/run_inference.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+
+script_dir=$(readlink -f $(dirname "$0"))
+default_config_dir=$script_dir/../configs
+
+# Define args
+TEST_DATA_BW=$1
```

Comment:
for user facing scripts, I prefer to have option parsing laid out like this - https://stackoverflow.com/questions/192249/how-do-i-parse-command-line-arguments-in-bash

positional arguments can get confusing for the user, and with option parsing it's easier to add more options if needed later

Reply:
Looks like there are multiple methods in that link. Which one?

Reply:
I usually use method 1, but either work

Reply:
done.

Reply:
@ntadimeti do you also need to change the example run_inference.sh command in the README?

Reply:
I'll change it

Reply:
done !